test/tests/functional/pbs_reservations.py

```diff
@@ -51,6 +51,19 @@ class TestReservations(TestFunctional):
     reservations
     """
 
+    def setUp(self):
+        super().setUp()
+        self._submitted_reservations = []
+        # self.server.manager(MGR_CMD_SET, SERVER, {'log_events': '8191'})
+
```

Comment:
To get list of reservations you could use: self.server.status(RESV)
(or) self.server.reservations.
But in this case setup and teardown class are not required, because reservation will be deleted in teardown or in setup class of next testcase. 
If it is not cleaned up then we can explicitly delete the reservation in teardown.

Reply:
As I recall, and this was quite some time ago, if the reservations created during the test were not removed early in the teardown of the test, some of the tests would would take a very long time (minutes) during teardown.  I was frequently run one such test, so the delay became annoying.  By removing the reservations early in the teardown process, the delay was avoided.  Perhaps there is a better place to perform the removal.  Suggestions are welcome.